pytest/tests/sanity/network_drop_package.py

```diff
@@ -16,26 +17,30 @@
 # Ratio of message that are dropped to simulate bad network performance
 DROP_RATIO = 0.05
 
+DONT_FILTER = ['Handshake', 'HandshakeFailure', 'LastEdge', 'Sync', 'RequestUpdateNonce', 'ResponseUpdateNonce']
```

Comment:
In stress.py packages_drop I only filter Handshake*, and it works fine. Are you sure you need to filter all of these?